### PR TITLE
Post patch: Building a Settings component

### PR DESCRIPTION
**Pull request by argyleink**
adds community submission 👍🏻

### CODE REVIEW

File: src/site/content/en/blog/building-a-settings-component/index.md [hunk 1]
```diff
@@ -964,4 +964,7 @@ to the [Community remixes](#community-remixes) section below!
 
 ## Community remixes
 
-<i>awaiting submissions!</i>
+- [@tomayac](https://twitter.com/tomayac) with a taste expression regardings the
```
Comment (mihajlija) [inline src/site/content/en/blog/building-a-settings-component/index.md:967]: Can you rephrase this? I don't really get what "a taste expression" means, and there's a typo in "regarding**s**"

Comment (argyleink) [inline src/site/content/en/blog/building-a-settings-component/index.md:967]:
```suggestion
- [@tomayac](https://twitter.com/tomayac) with their style regarding the
```

Comment (argyleink) [reply src/site/content/en/blog/building-a-settings-component/index.md:967]: thanks for the catch and change request @mihajlija, thoughts on this update?

**Comment by mihajlija** [comment]
LGTM, thanks for updating! :)

**Review by mihajlija — APPROVED** [review]
